client/my-sites/upgrades/checkout-thank-you/domain-mapping-details.jsx

```diff
@@ -9,6 +9,7 @@ import React from 'react';
 import { getDomainManagementUrl } from './utils';
 import { getPrimaryDomain, isSubdomain } from 'lib/domains';
 import { isPlan } from 'lib/products-values';
+import { MAP_EXISTING_DOMAIN, MAP_SUBDOMAIN } from 'lib/url/support';
```

Comment:
I'm not a fan of declaring global constants here even if they are limited to this module. I'd rather have `import support from 'lib/url/support';` and use `support.MY_CONST` everywhere so we understand immediately that this constant is related to the url support lib. Those kind of small conventions can really help us once we have multiple libs declaring constants imported in the same module I think